Code/EnginePlugins/GameComponentsPlugin/Gameplay/ProjectileComponent.h

```diff
@@ -83,6 +84,9 @@ class EZ_GAMECOMPONENTS_DLL ezProjectileComponent : public ezComponent
   /// If 0, the projectile is not affected by gravity.
   float m_fGravityMultiplier; // [ property ]
 
+  // Should a prefab be spawned when projectile is static
+  bool m_bSpawnPrefabOnStatic; // [ property ]
```

Comment:
I find this name a bit confusing. If I understand that correctly when set to true this means that the death prefab is spawned when the projectile doesn't move anymore (or only moves very slowly)? I can't think of a better name atm but maybe you could extent the comment to make it clearer what this does?

Reply:
Changed the comment